### DISCUSSION TITLE
Draw layer only between specific zoom levels

### DISCUSSION THREAD

**Discussion by demiantres** [mapsforge/vtm#872] [Q&A] [answered]
Hi,

Is there already built in functionality to limit a layer to specific zoom ranges? For instance, I want to draw a layer containing markers only between zoom levels 5 and 10.

**Comment by devemux86** [comment, marked as answer]
Apps are responsible for such custom workflows.
You can remove / (re) add or enable layers in the needed zooms.

**Reply by demiantres** [reply]
Ok, thanks!
